ui/panels-plugin/src/plugins/bar-chart/BarChartPanel.tsx

```diff
@@ -31,6 +31,7 @@ export function BarChartPanel(props: BarChartPanelProps) {
   const PADDING = chartsTheme.container.padding.default;
 
   const { queryResults, isLoading, isFetching } = useDataQueries('TimeSeriesQuery'); // gets data queries from a context provider, see DataQueriesProvider
+  const { queryResults: queryResults2, isLoading: isLoading2, isFetching: isFetching2 } = useDataQueries('TraceQuery'); // gets data queries from a context provider, see DataQueriesProvider
```

Comment:
I'd prefer traceQueryResult and change the first one to timeSeriesQueryResult.

Other wise when we have logs we will have queryResults3 and so on.